test_regress/driver.pl

```diff
@@ -961,6 +963,7 @@ sub compile_vlt_flags {
     unshift @verilator_flags, "-CFLAGS -fsanitize=address,undefined -LDFLAGS -fsanitize=address,undefined" if $param{sanitize};
     unshift @verilator_flags, "--make gmake" if $param{verilator_make_gmake};
     unshift @verilator_flags, "--make cmake" if $param{verilator_make_cmake};
+    unshift @verilator_flags, "--valgrind" if $opt_valgrind;
```

Comment:
Any unknown flag is passed through, so no special decoding of valgrind flag is needed by driver.pl.